tests/integration_test.rs

```diff
@@ -2071,65 +2299,59 @@ fn test_all_modifiers() {
         .build();
 
     let mut tracer = StateSaverTracer::default();
-    let (result, _) = run_program(&bin_path, vm_custom, &mut [Box::new(&mut tracer)]);
+    let output = run_program(&bin_path, vm_custom, &mut [Box::new(&mut tracer)]);
     let vm_final_state = tracer.state.last().unwrap();
+    let result = output.storage_zero;
     assert_eq!(result, U256::MAX - U256::from(8 - 4) + 1); // U256::MAX+1 == 2**256
     assert!(vm_final_state.flag_lt_of && vm_final_state.flag_eq && !vm_final_state.flag_gt);
 }
 
-#[test]
-fn test_near_call_panics_with_label() {
-    let bin_path = make_bin_path_asm("near_call_panics_with_label");
-    let vm = VMStateBuilder::default().build();
-    let (result, _) = run_program(&bin_path, vm, &mut []);
-    assert_eq!(result, U256::from(7));
-}
-
```

Comment:
Why was this test removed?

Reply:
It was not removed, only moved to another place (line 2251).